src/createPackageJson.js

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
Convention says that this file name should be create-package-json.js

https://github.com/felixge/node-style-guide/pull/36/commits